### ISSUE TITLE
Web frontend and scripts generating JSON?

### ISSUE THREAD

**Issue by ffainelli** [nluedtke/linux_kernel_cves#11]
Hi,

I would very much like to replicate the web frontend on a local site as well as be able to include additional remotes/branches (mostly downstream kernel forks) to the list of kernels affected/fixed. Do you have any plans to make the web frontend as well as the tools used to produce the json files available in the near future?

Thanks a lot for doing this!

**Comment by nluedtke** [comment]
I am still in process of decoupling the generation of the data from some pen-testing/security libraries that I don't intend to release. Once that is done, I do plan to have the code available. 

The Web Frontend is a different story. I reached out to my web programmer @quietcorey and I believe he will add the code for the frontend once he has a chance to lint it. 

What downstream forks are you looking for? Some of those are checked but not exposed by default, if there is a enough interest we could just add it to the regular daily update.

**Comment by ffainelli** [comment]
> I am still in process of decoupling the generation of the data from some pen-testing/security libraries that I don't intend to release. Once that is done, I do plan to have the code available.

OK, great, thanks!

> 
> The Web Frontend is a different story. I reached out to my web programmer @quietcorey and I believe he will add the code for the frontend once he has a chance to lint it.

That would be great!

> 
> What downstream forks are you looking for? Some of those are checked but not exposed by default, if there is a enough interest we could just add it to the regular daily update.

We maintain forks of the stable Linux kernel for STB/CM chips here: https://github.com/Broadcom/stblinux-4.9. Internally these trees contain the full kernel.org history plus our own downstream commits, but externally we only publish flat tarballs that take out the kernel.org history. I would assume other vendors (e.g: Google's ChromeOS, other SoC vendors) might have similar situations.

**Comment by nluedtke** [comment]
The Web Frontend code is now in the repo. I will make a separate issue for the generation code.

**Comment by ffainelli** [comment]
Thanks!
